static/astro.config.mjs

```diff
@@ -57,15 +59,8 @@ export default defineConfig({
       })
     ),
     tasks(),
-    import.meta.env.PROD &&
```

Comment:
What's causing the miss compile? Is the column selector in the css when you run npm run build ?

Playform compression does images, js, css, and html. Does this do that too?

What's the usage and track record of each?

Trading a dependency that we've used for a while needs a little why.

Reply:
<img width="1009" alt="Captura de pantalla 2024-10-24 a la(s) 2 42 37 a m" src="https://github.com/user-attachments/assets/0ac0687f-3852-448f-a8c4-e85d2b147d76">
Fixed, put the compress on the last line

Reply:
Sorry, I will be more careful with the changes I make, I will inform you first before making any changes.